contribute/contribute-pull-requests/add_hook.md

```diff
@@ -0,0 +1,14 @@
+# How To add a new Hook
+Adding a new hook is quite a simple process. It can be done in only tree steps.
+
+## 1) Add a call to Hook::exec()
+
+As you can see on this [PR](https://github.com/PrestaShop/PrestaShop/pull/34431/files) Adding a simple Hook::exex() statement is enought to add a new hook. In order for it to be complete, there is a couple more steps to do.
```

Comment:
`exec`

Reply:
I would be more precise here. In the shown example, you don't write about "adding a hook" but adding code that executes the hook in this place. That doesn't mean the hook is fully added to the system if you add this code.

Reply:
This is the only way I was told how to do it maybe you can enlight me ?

Reply:
I think @kpodemski means that adding the `Hook::exec()` (part 1) will trigger the hook, but if you dont do the parts 2 and parts 3 it will work only partially